src/trans-word-level/property.cpp

```diff
@@ -696,6 +696,7 @@ void property(
   auto obligations = property_obligations(property_expr, no_timeframes);
 
   // Map obligations onto timeframes.
+  prop_handles.clear();
   prop_handles.resize(no_timeframes, true_exprt());
```

Comment:
Since this is now always cleared shouldn't this be returned by value rather than being required as argument?

Reply:
Yes, will action.

Reply:
Will be separate PR.

Reply:
#934